services/victoriametrics/victoriametrics.go

```diff
@@ -0,0 +1,336 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package victoriametrics provides facilities for working with VictoriaMetrics.
+package victoriametrics
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"time"
+
+	"github.com/percona/pmm/utils/pdeathsig"
+	config "github.com/percona/promconfig"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"gopkg.in/reform.v1"
+	"gopkg.in/yaml.v2"
+
+	"github.com/percona/pmm-managed/models"
+	"github.com/percona/pmm-managed/services/prometheus"
+)
+
+const updateBatchDelay = 3 * time.Second
+
+var (
+	enabled       = os.Getenv("PERCONA_TEST_VM") == "1" || os.Getenv("PERCONA_TEST_VM") == "true"
```

Comment:
And what if it is "TRUE"? Please use strconv.ParseBool as in other pmm-managed code

Reply:
change dto strconv.Bool